packages/grid/_modules_/grid/components/panel/Panel.tsx

```diff
@@ -2,46 +2,18 @@ import * as React from 'react';
 import ClickAwayListener from '@material-ui/core/ClickAwayListener';
 import Paper from '@material-ui/core/Paper';
 import Popper from '@material-ui/core/Popper';
-import { makeStyles, Theme } from '@material-ui/core/styles';
 import { ApiContext } from '../api-context';
 import { isMuiV5 } from '../../utils';
-
-const useStyles = makeStyles(
```

Comment:
Why move the style outside of the panel? I think that it would be much better to keep them as close as possible to the underlying nodes. As far as I remember, the only reason why we have a large CSS file at the root is to improve the performance when rendering the cells. This is not a concern with the panel as only opened temporarily.

Reply:
This is to be able to reuse the css classes outside the root panel. 
This is used in `/story/x-grid-demos-custom-components--custom-panel`

Current issue with field
![image](https://user-images.githubusercontent.com/936978/105475527-bd467980-5c9f-11eb-90a2-027f515dcb14.png)

Reply:
I don't understand, what's the link?

Reply:
I just fixed it, I couldn't find the style breaking it.